assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -117,57 +117,69 @@ export const plotSeriesSelectTable = function (elem,
         .attr('tabindex', 0)
         .attr('role', 'listbox');
 
-
     table.append('thead')
         .append('tr')
-            .selectAll('th')
-            .data(columnHeaders)
-            .enter().append('th')
-                .attr('scope', 'col')
-                .text(d => d);
+        .selectAll('th')
+        .data(columnHeaders)
+        .enter().append('th')
+        .attr('scope', 'col')
+        .text(d => d);
 
     table.append('tbody')
+        .attr('class', 'usa-fieldset')
         .selectAll('tr')
         .data(availableParameterCodes)
         .enter().append('tr')
-            .attr('ga-on', 'click')
-            .attr('ga-event-category', 'selectTimeSeries')
-            .attr('ga-event-action', (parm) => `time-series-parmcd-${parm.parameterCode}`)
-            .attr('role', 'option')
-            .classed('selected', parm => parm.selected)
-            .attr('aria-selected', parm => parm.selected)
-            .on('click', function (parm) {
-                if (!parm.selected) {
-                    store.dispatch(Actions.updateIVCurrentVariableAndRetrieveTimeSeries(siteno, parm.variableID));
-                }
-            })
-            .call(tr => {
-                let parmCdCol = tr.append('th')
-                    .attr('scope', 'row');
-                parmCdCol.append('span')
-                    .text(parm => parm.description)
-                    .call(appendTooltip, parm => `Parameter code: ${parm.parameterCode}`);
-                tr.append('td')
-                    .append('svg')
-                    .attr('width', SPARK_LINE_DIM.width.toString())
-                    .attr('height', SPARK_LINE_DIM.height.toString());
-                tr.append('td')
-                    .text(parm => parm.timeSeriesCount);
-                tr.append('td')
-                    .style('white-space', 'nowrap')
-                    .text(parm =>`${config.uvPeriodOfRecord[parm.parameterCode].begin_date} to ${config.uvPeriodOfRecord[parm.parameterCode].end_date}`);
-            });
+        .attr('ga-on', 'click')
+        .attr('ga-event-category', 'selectTimeSeries')
+        .attr('ga-event-action', (param) => `time-series-parmcd-${param.parameterCode}`)
+        .attr('role', 'option')
+        .classed('selected', param => param.selected)
+        .attr('aria-selected', param => param.selected)
+        .on('click', function (param) {
+            if (!param.selected) {
+                store.dispatch(Actions.updateIVCurrentVariableAndRetrieveTimeSeries(siteno, param.variableID));
+            }
+        })
+        .call(tr => {
+
+            let paramSelectCol = tr.append('td')
+                .attr('scope', 'row');
```

Comment:
There should only be one element in a row with 'scope' set to 'row'. This one should probably go since, the radio button itself doesn't really identify the row.

Reply:
👍